common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -753,13 +754,24 @@ private void validateFrameworkDetails(Request request) {
       Map<String, Object> framework =
           (Map<String, Object>) request.getRequest().get(JsonKey.FRAMEWORK);
       if (!MapUtils.isEmpty(framework)) {
-        String frameworkId = (String) framework.get(JsonKey.ID);
-        if (StringUtils.isBlank(frameworkId)) {
-          throw new ProjectCommonException(
-              ResponseCode.mandatoryParamsMissing.getErrorCode(),
-              ResponseCode.mandatoryParamsMissing.getErrorMessage(),
-              ERROR_CODE,
-              StringFormatter.joinByDot(JsonKey.FRAMEWORK, JsonKey.ID));
+        if (framework.get(JsonKey.ID) instanceof List) {
+          List<String> frameworkId = (List<String>) framework.get(JsonKey.ID);
+          if (frameworkId.size() > 1 || CollectionUtils.isEmpty(frameworkId)) {
+            throw new ProjectCommonException(
+                ResponseCode.dataSizeError.getErrorCode(),
```

Comment:
Improper error response code is incorrect.

Maximum upload data size should be {0}